package.json

```diff
@@ -81,8 +81,8 @@
     "zx": "^4.2.0"
   },
   "peerDependencies": {
-    "@stripe/connect-js": "^2.0.1",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "@stripe/connect-js": "^2.0.1 || ^2.0.1-beta.1",
```

Comment:
The GA and beta packages should be kept separate, we don't want to encourage mixing the packages.

Reply:
Thank you for the suggestion! Just committed setting the `@stripe/connect-js`'s version to a GA one